packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/info.jsx

```diff
@@ -0,0 +1,125 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Popover, Money, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+
+const commission_tooltip_margin = 35;
+const stop_out_tooltip_margin = 135;
+
+const MultipliersInfo = ({
+    amount = 0,
+    commission = 0,
+    className,
+    currency,
+    multiplier = 0,
+    is_tooltip_relative,
+    should_show_tooltip,
+    should_show_percentage_tooltip,
+    stop_out = 0,
+}) => {
+    const commission_text = (
+        <Text
+            as='p'
+            line_height='s'
+            size='xxs'
+            className={classNames({
+                [`${className}-tooltip-text`]: className,
+            })}
+        >
+            <Localize
+                i18n_default_text='Commission <0/>'
+                components={[<Money key={0} amount={commission} currency={currency} show_currency />]}
+            />
+        </Text>
+    );
+
+    const stop_out_text = (
+        <Text
+            as='p'
+            line_height='s'
+            size='xxs'
+            className={classNames({
+                [`${className}-tooltip-text`]: className,
+            })}
+        >
+            <Localize
+                i18n_default_text='Stop out <0/>'
+                components={[<Money key={0} amount={stop_out} currency={currency} show_currency />]}
+            />
+        </Text>
+    );
+
+    const commission_tooltip = (
+        <Localize
+            i18n_default_text='<0>{{commission_percentage}}%</0> of (<1/> * {{multiplier}})'
+            values={{
+                commission_percentage: Number((commission * 100) / (multiplier * amount)).toFixed(4),
```

Comment:
Do we get `Infinity` if multiplier is **0**? Is this a possible case?

Reply:
No, `0` multipliers shouldn't be possible. 😄